### PR TITLE
[ISSUE-169] fix: us link account form

### PR DESCRIPTION
**Pull request by jjramirezn**
US account details form was impossible to advance due to wrong error handling and clearing, this is now fixed. Also modified a little the handling of errors for create-external-account, because on error the API doesn't return a json so it was always failing when trying to parse it

### CODE REVIEW

File: src/app/api/bridge/external-account/create-external-account/route.ts [hunk 1]
```diff
@@ -65,31 +65,40 @@ export async function POST(request: NextRequest) {
...
+                        headers: {
+                            'Content-Type': 'application/json',
+                        },
+                    })
+                }
+            } catch (e) {
+                console.error('Error creating external account', e)
             }
-            throw new Error(`HTTP error! status: ${response.status}`)
+            return new NextResponse('', {
```
Comment (panosfilianos) [inline src/app/api/bridge/external-account/create-external-account/route.ts:94]: question (non-blocking): is an empty data field here enough to handle all errors to function it returns it too? wdyt?

**Review by panosfilianos — APPROVED** [review]

File: src/app/api/bridge/external-account/create-external-account/route.ts (hunk 1 above)
Comment (panosfilianos) [reply src/app/api/bridge/external-account/create-external-account/route.ts:94]: as an additional note that's visible in the TODOs in these diffs: we need a std way to handle these errors and propagate that error state in the whole UI. I'm currently creating a prototype of what a solution could look like in `peanut-wallet`.

tl;dr: don't waste too much time on this

Comment (jjramirezn) [reply src/app/api/bridge/external-account/create-external-account/route.ts:94]: the empty data field is not enough and I love that you are doing something about the error state, it is needed

Comment (panosfilianos) [reply src/app/api/bridge/external-account/create-external-account/route.ts:94]: cool!
this is not the scope to fix that and I have a Task and TODOs around that.
